sources/packages/backend/libs/queue/src/model/queue.model.ts

```diff
@@ -0,0 +1,4 @@
+export interface QueueModel {
```

Comment:
Bull has its own type `BullModuleOptions`. Was the `QueueModel` created to add the `readonly` option?